utilities_for_test.go

```diff
@@ -154,17 +155,14 @@ func NewIntegrationTestEnv(t *testing.T) IntegrationTestEnv {
 			SetInitialBalance(NewHbar(150)).
 			SetAutoRenewPeriod(time.Hour*24*81 + time.Minute*26 + time.Second*39).
 			Execute(env.Client)
-		if err != nil {
-			panic(err)
-		}
+
+		require.NoError(t, err)
 
 		receipt, err := resp.SetValidateStatus(true).GetReceipt(env.Client)
-		if err != nil {
-			panic(err)
-		}
+		require.NoError(t, err)
 
-		env.OriginalOperatorID = env.Client.GetOperatorAccountID()
-		env.OriginalOperatorKey = env.Client.GetOperatorPublicKey()
+		// env.OriginalOperatorID = env.Client.GetOperatorAccountID()
+		// env.OriginalOperatorKey = env.Client.GetOperatorPublicKey()
```

Comment:
why do we have to leave these two commented lines here?

Reply:
My bad, forgot to remove them after running the tests. Thanks